packages/test-eslint-config/package.json

```diff
@@ -10,6 +10,9 @@
     "eslint": "^8.28.0"
   },
   "eslintConfig": {
-    "extends": "@mediamonks/eslint-config"
+    "extends": "@mediamonks/eslint-config",
+    "parserOptions": {
```

Comment:
Do we also need this in the docs, or was this only for the `#private` stuff in the test code?